### DISCUSSION TITLE
Panopticon in relation to RS Firewall

### DISCUSSION THREAD

**Discussion by ReneSchouten62** [akeeba/panopticon#795] [General]
Has anyone installed Panopticon and are also using RSFirewall. I'm just curious how this got started.

**Comment by nikosdion** [comment]
Your issue has nothing to do with RSFirewall. Your issue can be addressed following 
[the connection troubleshooting guide](https://github.com/akeeba/panopticon/wiki/Connection-Troubleshooting).

Hint: which core Joomla plugins need to be enabled for Panopticon to work?
